nemo_skills/evaluation/eval_group/bfcl.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# BFCLv3 evaluation group
+# Combines the single turn and multi turn ASTs - total of 17 subsets of BFCLv3
+
+score_module: nemo_skills.evaluation.eval_group.bfcl_score
+
+jobs:
+  # Simple AST
+  - name: bfcl.simple 
```

Comment:
it's probably better to list them under a single job, with name: bfcl and then all benchmarks comma-separated. eval_group should work fine with that, and if you do this, then people can group them together on fewer nodes with num_jobs argument. But in the current way, they will always run on separate nodes